createmobilespec/createmobilespec.js

```diff
@@ -71,7 +71,12 @@ function pluginAdd(pluginName, searchPath, extraFlags) {
     if (extraFlags) {
         command += extraFlags;
     }
-    shelljs.exec(command);
+    executeShellCommand(command);
+}
+
+function executeShellCommand(command) {
+    console.log('$ ' + command);
+    return shelljs.exec(command);
```

Comment:
```suggestion
    shelljs.exec(command);
```

I _think_ returning the result of `shelljs.exec()` call would be useless.

Reply:
https://github.com/apache/cordova-mobile-spec/blob/3e30f761c342bb93378686f67f8707437d7a1b7b/createmobilespec/createmobilespec.js#L691 uses the return value.

Reply:
Got it 👍